### PR TITLE
Update dataset link

### PR DESCRIPTION
**Pull request by wgifford**
Update dataset links.
Update model link due to renaming of HF model page.

### CODE REVIEW

**Review by fayvor — APPROVED** [review]
Looks good!

**Review by fayvor — COMMENTED** [review]
How does this work? Does pandas.read_csv() recognize this protocol?

File: recipes/Time_Series/Preprocessor_Use_and_Performance_Evaluation.ipynb [hunk 1]
```diff
@@ -113,7 +112,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA_FILE_PATH = \"~/Downloads/energy_dataset.csv.zip\""
+    "DATA_FILE_PATH = \"hf://datasets/vitaliy-sharandin/energy-consumption-hourly-spain/energy_dataset.csv\""
```
Comment (fayvor) [inline recipes/Time_Series/Preprocessor_Use_and_Performance_Evaluation.ipynb:115]: How does this work? Does pandas.read_csv() recognize this protocol?

Comment (wgifford) [reply recipes/Time_Series/Preprocessor_Use_and_Performance_Evaluation.ipynb:115]: Apparently, yes it does. The "hf://" syntax is directly from the HF dataset page when you choose "use this dataset".

**Pull request merged**